File: examples-sdk/zig/203-llm-code-review-with-tools/main.zig
// 203-llm-code-review-with-tools — Zig SDK equivalent
// Compile: vil compile --from zig --input 203-llm-code-review-with-tools/main.zig --release

const vil = @import("vil");

pub fn main() void {
    var server = vil.Server.init("llm-code-review-tools", 3102);
    var code_review = vil.Service.init("code-review");
    code_review.endpoint("POST", "/code/review", "code_review_handler");
    server.service(&code_review);
    server.compile();
}

File: examples-sdk/go/203-llm-code-review-with-tools/main.go
// 203-llm-code-review-with-tools — Go SDK equivalent
// Compile: vil compile --from go --input 203-llm-code-review-with-tools/main.go --release
package main

import vil "github.com/OceanOS-id/vil-go"

func main() {
	s := vil.NewServer("llm-code-review-tools", 3102)

	code_review := vil.NewService("code-review")
	code_review.Endpoint("POST", "/code/review", "code_review_handler")
	s.Service(code_review)

	s.Compile()
}
